modules/apps/remote-app/remote-app-service/src/main/java/com/liferay/remote/app/service/impl/RemoteAppEntryLocalServiceImpl.java

```diff
@@ -95,6 +96,24 @@ public RemoteAppEntry addCustomElementRemoteAppEntry(
 			String properties, String sourceCodeURL)
 		throws PortalException {
 
+		return addCustomElementRemoteAppEntry(
```

Comment:
Although it is not the best, right now as this API is evolving continuously, we are not overloading the methods and just adding the new parameters and updating the usages.

Be careful because I think this method is used by the site-initializer-extender module